### DISCUSSION TITLE
Do checkboxes work with default values?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [react-hook-form/react-hook-form#7808] [Q&A] [answered]
```  
 {
        defaultValues: {
            assigned: true,
        },
    });
 
 <input
                    className="ml-auto h-5 w-5"
                    type="checkbox"
                    id="assigneed"
                    {...register("assigned")}
/>
```

The checkbox is always unchecked at the beginning. This isn't the behaviour I would expect, and there are no issues on this topic so I have no idea what is happening.

**Comment by Moshyfawn** [comment, marked as answer]
Works as expected: see [this CSB](https://codesandbox.io/s/prod-leaf-302fz?file=/src/App.tsx)

**Reply by Nikola-Milovic** [reply]
Wasn't working in 7.14.2 for me. Updating to 7.27.0 and it's working as intended. Thanks!

**Reply by Moshyfawn** [reply]
[Can't reproduce](https://codesandbox.io/s/hidden-microservice-51svw?file=/src/App.tsx) with 7.14.2 either :P
Anyway, I'm glad it's working for you now
